CC-Bootloader/rfcat_bootloader

```diff
@@ -7,31 +7,31 @@ from serial.serialutil import SerialException
 
 
 bootloader_error_codes = {
-  '0' : "OK",
-  '1' : "Intel HEX Invalid",
-  '2' : "Bad Checksum",
-  '3' : "Bad Address",
-  '4' : "Bad Record Type",
-  '5' : "Record Too Long"
+  b'0' : "OK",
```

Comment:
i appreciate the help!
two requests:
1) please split these out into two separate pull requests (when i'm contributing code, i'll create a branch in my own fork for each PR, each based on the upstream master branch.  this allows each to be evaluated and merged as "bite-sized chunks"
2) reach out to @roaldnefs , who has volunteered to create a PR for porting RfCat to use Python3 (since much of the code deltas in this PR are for Python3)

i realize this may take a little doing on your end, and i apologize.